src/navs/index.js

```diff
@@ -1995,54 +1979,45 @@ export const docsMenu = {
                     icon: 'Home',
                     color: 'seagreen',
                 },
-                /*
                 {
-                    name: 'Getting started',
-                    url: '/docs/feature-flags/start',
-                    icon: 'GraduationCap',
-                    color: 'red',
-                },
-                */
-                {
-                    name: 'Product manual',
-                    url: '/docs/feature-flags/manual',
+                    name: 'Installation',
+                    url: '/docs/feature-flags/installation',
                     icon: 'Book',
                     color: 'blue',
                 },
                 {
-                    name: 'Features',
+                    name: 'Creating feature flags',
+                    url: '/docs/feature-flags/creating-feature-flags',
+                    icon: 'Flag',
+                    color: 'orange',
                 },
                 {
-                    name: 'Bootstrapping & local evaluation',
-                    url: '/docs/feature-flags/bootstrapping-and-local-evaluation',
-                    icon: 'Bolt',
+                    name: 'Adding your code',
+                    url: '/docs/feature-flags/adding-feature-flag-code',
+                    icon: 'Code',
                     color: 'salmon',
                 },
                 {
-                    name: 'Rollout strategies',
-                    url: '/docs/feature-flags/rollout-strategies',
-                    icon: 'Percentage',
-                    color: 'orange',
+                    name: 'Features',
                 },
                 {
-                    name: 'Multivariate feature flags',
-                    url: '/docs/feature-flags/multivariate-flags',
-                    icon: 'Flag',
-                    color: 'purple',
+                    name: 'Server-side local evaluation',
+                    url: '/docs/feature-flags/local-evaluation',
+                    icon: 'Bolt',
+                    color: 'salmon',
                 },
                 {
-                    name: 'Payloads',
-                    url: '/docs/feature-flags/payloads',
-                    icon: 'Brackets',
-                    color: 'blue',
+                    name: 'Client-side bootstrapping',
+                    url: '/docs/feature-flags/local-evaluation',
```

Comment:
```suggestion
                    url: '/docs/feature-flags/bootstrapping',
```

Wrong URL

Reply:
It was updated in a later PR, but nice catch!